terraform/api.tf

```
@@ -34,16 +42,18 @@ module "api" {
     DJANGO_DANDI_DANDISETS_EMBARGO_LOG_BUCKET_NAME = module.sponsored_embargo_bucket.log_bucket_name
     DJANGO_DANDI_DOI_API_URL                       = "https://api.datacite.org/dois"
     DJANGO_DANDI_DOI_API_USER                      = "dartlib.dandi"
-    DJANGO_DANDI_DOI_API_PREFIX                    = "10.48324"
+    DJANGO_DANDI_DOI_API_PREFIX                    = "10.${var.dandi_instance_datacite_id}"
     DJANGO_DANDI_DOI_PUBLISH                       = "true"
     DJANGO_SENTRY_DSN                              = data.sentry_key.this.dsn_public
     DJANGO_SENTRY_ENVIRONMENT                      = "production"
     DJANGO_CELERY_WORKER_CONCURRENCY               = "4"
-    DJANGO_DANDI_WEB_APP_URL                       = "https://dandiarchive.org"
-    DJANGO_DANDI_API_URL                           = "https://api.dandiarchive.org"
-    DJANGO_DANDI_JUPYTERHUB_URL                    = "https://hub.dandiarchive.org/"
+    DJANGO_DANDI_WEB_APP_URL                       = var.dandi_instance_url
+    DJANGO_DANDI_API_URL                           = "https://api.${var.dandi_instance_tld}"
+    DJANGO_DANDI_JUPYTERHUB_URL                    = "https://hub.${var.dandi_instance_tld}"
     DJANGO_DANDI_DEV_EMAIL                         = var.dev_email
-    DJANGO_DANDI_ADMIN_EMAIL                       = "info@dandiarchive.org"
+    DJANGO_DANDI_ADMIN_EMAIL                       = "info@${var.dandi_instance_tld}"
+    DANDI_INSTANCE_NAME                            = var.dandi_instance_name 
+    DANDI_DATACITE_DOI_ID                          = var.dandi_instance_datacite_id
```

```suggestion
    DJANGO_DANDI_INSTANCE_NAME                            = var.dandi_instance_name 
    DJANGO_DANDI_DATACITE_DOI_ID                          = var.dandi_instance_datacite_id
```
and accompany with mirrored changes to 
https://github.com/dandi/dandi-archive/blob/master/dandiapi/settings.py#L88-L93